COD60275.TXT

```diff
@@ -0,0 +1,260 @@
+OBJECT Codeunit 60275 Xml Dom Mgt.
+{
+  OBJECT-PROPERTIES
+  {
+    Date=01-09-17;
+    Time=13:21:54;
+    Version List=;
+  }
+  PROPERTIES
+  {
+    OnRun=BEGIN
+          END;
+
+  }
+  CODE
+  {
+    VAR
+      NpXmlTemplate2@6150619 : Record 6151551;
+      OutputTempBlob@6150613 : TEMPORARY Record 99008535;
+      ResponseTempBlob@6150626 : TEMPORARY Record 99008535;
+      RecRef@6150617 : RecordRef;
+      OutputOutStr@6150630 : OutStream;
+      ResponseOutStr@6150632 : OutStream;
+      Window@6150615 : Dialog;
+      PrimaryKeyValue@6150616 : Text;
+      BatchCount@6150629 : Integer;
+      HideDialog@6150621 : Boolean;
+      Initialized@6150622 : Boolean;
+      OutputInitialized@6150631 : Boolean;
+
+    LOCAL PROCEDURE "--- Init Xml"@6014400();
+    BEGIN
+    END;
+
+    PROCEDURE InitDoc@6150639(VAR XmlDoc@6150613 : DotNet "'System.Xml, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.Xml.XmlDocument";RootName@6150615 : Text);
```

Comment:
Either make this procedure local or remove the dotnet types from the parameter list - cannot be called from extensions.